### PR TITLE
Add IRB to cli options

### LINKED ISSUE #467: CLI to build environment with irb
Would you consider...

**Is your feature request related to a problem? Please describe.**
During testing, it is often easier to use a terminal to test certain functions or logic quickly. I generally load up some local variables and iterate over certain sections. The script runner is great but it isnt a direct replacement for a terminal. In fact, the script runner would complement building in script-runner while a terminal can be used to test out the portions as they are run.

**Describe the solution you'd like**
The cli seems like a great place for a command to build a irb terminal session using a container image that has a system and environment built based on the current configs that are loaded. That way you have full access to the definition database and can run quick tests.

### PR DESCRIPTION
**Pull request by jmthomas**
closes #467

### CODE REVIEW

File: openc3.sh [hunk 1]
```diff
@@ -34,14 +34,14 @@ case $1 in
     # This allows tools running in the container to have a consistent path to the current working directory.
     # Run the command "ruby /openc3/bin/openc3cli" with all parameters starting at 2 since the first is 'openc3'
     args=`echo $@ | { read _ args; echo $args; }`
-    docker run --rm --env-file "$(dirname -- "$0")/.env" --user=$OPENC3_USER_ID:$OPENC3_GROUP_ID -v `pwd`:/openc3/local:z -w /openc3/local $OPENC3_REGISTRY/openc3inc/openc3-operator:$OPENC3_TAG ruby /openc3/bin/openc3cli $args
+    docker run -it --rm --env-file "$(dirname -- "$0")/.env" --user=$OPENC3_USER_ID:$OPENC3_GROUP_ID -v `pwd`:/openc3/local:z -w /openc3/local $OPENC3_REGISTRY/openc3inc/openc3-operator:$OPENC3_TAG ruby /openc3/bin/openc3cli $args
```
Comment (jmthomas) [inline openc3.sh:37]: I think it's ok to always do this but not sure. The other commands still work.

File: openc3/bin/openc3cli [hunk 2]
```diff
@@ -526,6 +529,10 @@ if not ARGV[0].nil? # argument(s) given
   # Handle each task
   case ARGV[0].downcase
 
+  when 'irb'
+    ARGV.clear
+    IRB.start
```
Comment (jmthomas) [inline openc3/bin/openc3cli:534]: Have to clear ARGV first or IRB think it should interpret those as files to run

**Comment by jmthomas** [comment]
If you think this is worthwhile I'll add it to the rest of the projects

File: openc3.sh (hunk 1 above)
Comment (ryanmelt) [reply openc3.sh:37]: Nevermind, I think this is ok.  I'm pretty sure it was something in Github actions where I had to remove -it

**Pull request merged**